tests/foreman/api/test_puppetmodule.py

```diff
@@ -1,125 +0,0 @@
-"""Unit tests for the ``katello/api/v2/puppet_modules`` paths.
-
-:Requirement: Puppetmodule
-
-:CaseAutomation: Automated
-
-:CaseLevel: Acceptance
-
-:CaseComponent: Puppet
-
-:Assignee: vsedmik
-
-:TestType: Functional
-
-:CaseImportance: High
-
-:Upstream: No
-"""
-import pytest
-from nailgun import entities
-
-from robottelo.constants import PUPPET_MODULE_NTP_PUPPETLABS
```

Comment:
Should the constant be removed as well? Searching for it, it appears also in:

tests/foreman/api/test_hostgroup.py
tests/foreman/api/test_contentviewversion.py
tests/foreman/api/test_contentview.py

... in relation with puppet repositories, do we want to remove also the tests for puppet repos?

Reply:
this is just first PR, in the whole EPIC, 
@latran @vsedmik and I started to work on it. 
I see maybe it's time to send out an email about it.
We already identified I hope all 6.10 puppet failing tests which needs to be removed or fixed. 

Definitely nice catch up, maybe the constants would not be needed afterwards.